examples/multi-service-profile/README.md

```diff
@@ -0,0 +1,7 @@
+# CBR Multi Service Profile
+
+An end-to-end example that uses the module's default variable values. This example uses the IBM Cloud Provider to automate the following infrastructure::
+
+ - Create a VPC and create a CBR zone to allowlist the VPC.
+ - Create a service reference based CBR zone.
```

Comment:
I think we could just add some more specifics about the target services in this example. In particular, highlighting fact that there is an AND in the code on the resource group, and the optionally passed access tags.